grafana/Tiltfile

```diff
@@ -136,6 +146,12 @@ def grafana(context, plugin_files, grafana_version='latest', namespace='grafana'
     chart_values['image']['repository'] = image_repository
     chart_values['image']['tag'] = image_tag
 
+    # Sets the passed in additional Grafana_init fields
+    if len(extra_grafana_ini) > 0:
+        grafana_ini = chart_values["grafana.ini"] 
```

Comment:
question: Can this code be simpler because we're doing in-place modification?

I don't know the details of python / starlark dictionaries. Is the `update_grafana_ini` function doing an in-place modification of `chart_values["grafana.ini"]`? If not, when are keys and values copied? 

If there's no copy operation, this seems like it does: 
```
a = a
a = update(a, b) // a's values are updated in-place
a.update(a)
```

running a Python repl, I get:
```python
>>> a = {'one': 1, 'dict': {'two': 2}}
>>> a
{'one': 1, 'dict': {'two': 2}}
>>> b = a
>>> def update(arg1, arg2):
...     for k, v in arg2.items():
...             arg1[k] = v
...
>>> c = {'three': 3}
>>> update(b, c)
>>> a
{'one': 1, 'dict': {'two': 2}, 'three': 3}
>>> b
{'one': 1, 'dict': {'two': 2}, 'three': 3}
>>> c
{'three': 3}
```

which does seem to imply some in-place modification of the dictionary. I think that means you can simplify the code, since we aren't protecting data with copies.

Reply:
nice catch, thanks! simplified the code, updated the README and merged.